.github/actions/nexus-create-tag/action.yml

```diff
@@ -0,0 +1,39 @@
+name: Create nexus tag
+description: Create a tag in Nexus
+inputs:
+  tag-name:
+    description: The name of the tag to be created
+    required: true
+  nexus-username:
+    description: Nexus user name
+    required: true
+  nexus-password:
+    description: Nexus password
+    required: true
+  nexus-url:
+    description: Base URL to the Nexus server
+    required: false
+    default: "https://artifacts.alfresco.com/nexus"
+outputs:
+  tag-name-created:
+    description: The name of the tag created
+    value: ${{ steps.create-nexus-tag.outputs.tag-name-created }}
+
+runs:
+  using: composite
+  steps:
+    - name: create-nexus-tag
+      id: create-nexus-tag
+      shell: bash
+      env:
+        TAG_NAME: ${{ inputs.tag-name }}
+        NEXUS_USERNAME: ${{ inputs.nexus-username }}
+        NEXUS_PASSWORD: ${{ inputs.nexus-password }}
+        NEXUS_URL: ${{ inputs.nexus-url }}
+      run: |
+          echo "Creating tag $TAG_NAME in Nexus"
+          curl -u "${NEXUS_USERNAME}:${NEXUS_PASSWORD}" \
+            -X POST --header 'Content-Type: application/json' \
+            "${NEXUS_URL}/service/rest/v1/tags" \
+            -d "{\"name\":\"$TAG_NAME\"}"
+          echo "tag-name-created=$TAG_NAME" >> $GITHUB_OUTPUT
```

Comment:
I think you can simplify the output name to `tag`

Reply:
Update: @gionn mentioned, it probably not needed provide an output in this case because we already know the value which is the same passed as input.